qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -182,7 +182,7 @@ def get_raw(self) -> np.ndarray:
 
 class TimeAxis(Parameter):
     """
-    A simple :class:`Parameter` that holds the time interval
```

Comment:
Why this change. I think the correct way to do this is 
```
`:class:`.Parameter`
```

See the docs here for how this is resolved http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-roles